Frontend/ui-customer/src/Flow.purs

```diff
@@ -587,6 +587,8 @@ homeScreenFlow = do
   logField_ <- lift $ lift $ getLogFields
   (GlobalState currentState) <- getState
   void $ checkAndUpdateSavedLocations currentState.homeScreen
+  let isRideCompleted = getAndRemoveLatestNotificationType unit == "TRIP_FINISHED"
+  when isRideCompleted $ currentFlowStatus
```

Comment:
@prakharritik can you please explain the use case of this

Reply:
This is done to update view immediately in case of ride completed and user is on different screen. Normally screen will update after polling.